### PR TITLE
Add argument parsing

### PR DESCRIPTION
**Pull request by JacobLezberg**
- Implemented argparse
- Made example_inputs dir (copied in default example now just to test)
- auto-formatter did some spacing
- designated constants

### CODE REVIEW

File: popils.py [hunk 1]
```diff
@@ -1,33 +1,47 @@
...
+window_size = [WINDOW_WIDTH, WINDOW_HEIGHT]
 block_size = 50  # block size = 50px TODO dynamic block sizing
-screen = pygame.display.set_mode(size)
-done = False
+# initialize drawing surface
+screen = pygame.display.set_mode(window_size)
 
 
+def place_sub_gadget(code, bottom_row, col):
+    for i in range(GADGET_HEIGHT):
```
Comment (KirbyG) [inline popils.py]:
```suggestion
    for i in range(SUB_GADGET_HEIGHT):
```
recommend prepending SUB as the full gadget height would be 6 blocks

File: popils_constants.py [hunk 2]
```diff
@@ -9,12 +9,25 @@
...
 
-class VariableState(Enum):
-    UNUSED = 0
-    NEGATED = -1
-    NONNEGATED = 1
+GADGET_UNUSED = [LADDER, SUPPORT, LADDER, SUPPORT]
+GADGET_NEGATED = [LADDER, LADDER, SUPPORT, LADDER]
+GADGET_NONNEGATED = [SUPPORT, LADDER, LADDER, SUPPORT]
+SUB_GADGETS = [GADGET_UNUSED, GADGET_NEGATED, GADGET_NONNEGATED]
+GADGET_HEIGHT = 4  # in block units
```
Comment (KirbyG) [inline popils_constants.py]:
```suggestion
SUB_GADGET_HEIGHT = 4  # in block units
```
recommend prepending SUB as the full gadget height would be 6 blocks

**Review by KirbyG — CHANGES_REQUESTED** [review]
i'm not sure if it's standard practice to request small changes like this, but i kinda wanted to use the feature

**Comment by JacobLezberg** [comment]
> 
> 
> i'm not sure if it's standard practice to request small changes like this, but i kinda wanted to use the feature

Yeah, that's exactly what the feature is for :)
Much easier/cleaner to make changes in review than opening another branch to follow-up

**Review by KirbyG — APPROVED** [review]

**Pull request merged**